code/nacl-box/Hacl.NaCl.fst

```diff
@@ -16,6 +16,14 @@ module SS = Spec.SecretBox
 
 #set-options "--max_fuel 50 --max_fuel 0 --max_ifuel 0"
 
+[@@ Comment "Encrypt a message with a key and nonce.
```

Comment:
Should we add a comment saying that encryption and decryption can be done in-place?

Reply:
Good point. I'll add something :+1:

Reply:
Added in 239dc5c7ef2d81f9a47d0d296ca41b2b22c16404.